modules/asg/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "security_group_id" {
+  value = aws_security_group.private.id
+}
```

Comment:
Could you please, add a new line at the end of file.

Reply:
Added a new line.